includes/discovery/sensors/current/rfc1628.inc.php

```diff
@@ -34,6 +34,9 @@
     if (count($output_current) > 1) {
         $descr .= " Phase $index";
     }
+    if (is_array($data['upsOutputCurrent'])) {
```

Comment:
Other rfc1628 types have this code block:
```php
    if (is_array($data['upsOutputVoltage'])) {
        $upsOutputVoltage_value = $data['upsOutputVoltage'][0];
        $volt_oid .= ".0";
    }
```

You should probably update the oid being discovered as well.